### DISCUSSION TITLE
请教一下多运行商下的PaoPaoDNS相关设置

### DISCUSSION THREAD

**Discussion by KouTorisan** [kkkgo/PaoPaoGateWay#145] [问和答]
主：爱快 ip:192.168.0.1 (开启多线负载)

DNS1：PaoPaoDNS.CT(电信) IP.192.168.0.4
DNS2：PaoPaoDNS.CM(移动) IP.192.168.0.6

之前家里一直使用的单宽带电信，配合  PaoPaoDNS.CT(192.168.0.4)  以及  PaoPaoGateway  工作良好，但家里最近接入了移动宽带，想着统一使用  PaoPaoDNS.CT(192.168.0.4) ，但发现在指定移动宽带的线路下使用时返回的还是电信的相关CDN,甚至有时候返回的ip在移动线路下无法连接。所以又单独针对移动线路用  PaoPao-Pref  构建了  PaoPaoDNS.CM（192.168.0.6）。想请问是否有更好的一个DNS下就可以兼容方案？甚至是否有方案能够做到运行商线路并发，自动选择连接质量更好的运行商线路进行连接？

**Comment by kkkgo** [comment]
https://github.com/kkkgo/PaoPaoDNS/discussions/167
选择运营商线路是路由器的功能而不是dns的功能。不同运营商应该分开使用不同的dns。
如果你想两个运营商一起用，那么要么平时指定用某一个线路A，特殊应用或者流量分流到另一个线路B，要么主用某一个线路A，仅指定另一个线路B运营商IP段给线路B用。混合线路同时用的DNS不会有完美的解决方案，就比如你说的连接质量如何界定，ping延迟好不代表tcp延迟好，tcp延迟好不代表带宽大，这种操作在smartdns上也有，如果你想套娃你也可以试试。
